examples/kubernetes-local-redirect/lrp-svcmatcher.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumLocalRedirectPolicy
+metadata:
+  name: "lrp-svc"
+spec:
+  redirectFrontend:
+    serviceMatcher:
```

Comment:
nit: We could make this a little more consistent with other similar constructs in CNP:

```suggestion
    toServices:
```